web/api/backend_test.go

```diff
@@ -451,7 +451,7 @@ func TestHandlerTestCommand(t *testing.T) {
 	if err := json.Unmarshal(in, &raw); err != nil {
 		panic(err)
 	}
-	if raw["Oauth2l Response"] == "" {
+	if raw["Oauth2lResponse"] == "" {
```

Comment:
Key naming should follow camel case guidelines

Reply:
done!